lms/djangoapps/courseware/masquerade.py

```diff
@@ -182,6 +184,20 @@ def get_masquerading_group_info(user, course_key):
     return course_masquerade.group_id, course_masquerade.user_partition_id
 
 
+def get_masquerading_user_group(course_key, user, user_partition):
+    """
+    If the current user is masquerading as a generic student in a specific group, return that group
+    """
+    group_id, user_partition_id = get_masquerading_group_info(user, course_key)
```

Comment:
There's not a whole lot to this method, but I think it is worth adding unit test coverage to test_masquerade.py (for instance, see the unit test coverage for get_masquerading_group_info).